Adrian Filimon/lesson_7/lesson_7_3.py

```diff
@@ -0,0 +1,44 @@
+# Given a set of print methods:
+# print_registry = []
+#
+#
+# def greet(name):
+#     return "Greetings {}!".format(name)
+#
+#
+# def say_hello(name):
+#     return "Hello {}!".format(name)
+#
+#
+# def say_goodbye(name):
+#     return "Goodbye {}!".format(name)
+# Create a decorator called register that will update a list called print_registry with all the decorated functions names.
+
+print_registry = []
+
+
+def register(funct):
+    def inner_funct(*args):
```

Comment:
This code can be further simplified; is the inner function really needed?
Justify the answer, whichever that might be